src/ReactiveUI.Blend/ReactiveUI.Blend.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;uap10.0.16299;net5.0-windows10.0.19041;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net472;uap10.0.16299;net6.0-windows10.0.19041;netcoreapp3.1</TargetFrameworks>
```

Comment:
I removed netcoreapp3.1 from Splat as well I believe. Make tests target net6/net framework only.

Also net462 is becoming standard in 2 months so might as well set that as the base now as well.